src/lib/get-explorer-url.ts

```diff
@@ -0,0 +1,7 @@
+export const getExplorerUrl = (
```

Comment:
Nice! Can we also use this in deposit and footer? Either here on in a different PR

Reply:
I can do it here